.editorconfig

```diff
@@ -15,7 +12,7 @@ indent_size = 2
 trim_trailing_whitespace = true
 
 [*.cs]
-indent_size = 4
+indent_size = 5
```

Comment:
I triend tabs and i can get one lite more space. But if you don't want, you can, it's your project, not mine.